tests/specs/swap/swap.spec.ts

```diff
@@ -1,6 +1,6 @@
 import { test } from '../../fixtures/fixtures';
 
-const alexSdkPostRoute = 'https://gql.alexlab.co/v1/graphql';
+const alexSdkPostRoute = '*/**/v1/graphql';
```

Comment:
Shouldn't this reference the origin at all? Would `**alexlab.co/v1/graphql` work?

Reply:
pushing commit for this, think it's good to be specific to origin level at least (who knows maybe that could clash with another api)

Reply:
Tried a few combos before I got it to work while specifying origin

Reply:
Ok, thx for committing the change.